jdk/src/share/classes/java/security/Security.java

```diff
@@ -187,6 +195,18 @@ private static void initialize() {
             }
         }
 
+        // Load FIPS properties
+        if (loadedProps) {
+            boolean fipsEnabled = FIPSConfigurator.configureFIPS(props);
+            if (sdebug != null) {
+                if (fipsEnabled) {
+                    sdebug.println("FIPS mode enabled.");
+                } else {
+                    sdebug.println("FIPS mode disabled.");
+                }
+            }
+        }
+
```

Comment:
Please remove this blank line (that wasn't in the earlier pull request).

Reply:
removed